packages/core/useScriptTag/index.ts

```diff
@@ -0,0 +1,126 @@
+import { isString, MaybeRef, tryOnMounted, tryOnUnmounted } from '@vueuse/shared'
+import { ref } from 'vue-demi'
+import { ConfigurableDocument, defaultDocument } from '../_configurable'
+
+export interface UseScriptTagOptions {
```

Comment:
We can extend `ConfigurableDocument` here to have a single options object for configuration.